addons/website/views/snippets/s_card.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <template id="s_card" name="Card">
-    <div class="s_card o_card_img_top card o_cc o_cc1">
+    <div class="s_card o_card_img_top card o_cc o_cc1" data-vcss="001" data-vxml="001">
         <figure class="o_card_img_wrapper ratio ratio-16x9 mb-0">
             <img class="o_card_img card-img-top object-fit-cover" src="/web/image/website.s_card_default_image_1"/>
```

Comment:
A bit debatable 'cause it really depend by what the user wants to achieve 🤔... but I do agree that `container` should cover the most plausible scenarios.. ok, let's do that 👍 

Be aware that it will invalidate the alignment option, I'll add a condition for this as well 😉